phpBB/language/en/cli.php

```diff
@@ -94,6 +96,12 @@
 
 	'CLI_FIXUP_RECALCULATE_EMAIL_HASH_SUCCESS'	=> 'Successfully recalculated all email hashes.',
 
+	'CLI_MIGRATION_NAME'					=> 'Full migration name (you can replace the antislashs by slashs to avoid problems)',
```

Comment:
Is it name or path and name?
Rewrite: (use forward slashes instead of backslashes to avoid problems)